test/navbar-components/navigationBarTest.js

```diff
@@ -377,15 +377,18 @@ describe('navigationBar tests', function () {
         .waitUntil(() =>
           this.app.client
             .activateURLMode()
-            .waitForExist(urlbarIcon)
-            .getAttribute(urlbarIcon, 'class').then((classes) =>
+            .waitForExist(urlBarIcon)
+
+            // TODO: Fix
+            .getAttribute(urlBarIcon, 'class').then((classes) =>
               classes.includes('fa-unlock') && classes.includes('insecure-color')
```

Comment:
one of the test failures is due to that `insecure-color` has been removed on [this](https://github.com/brave/browser-laptop/pull/9299/files#diff-51a114fecaac13cd41162f4c956e7508L25) in favor of [this one](https://github.com/brave/browser-laptop/pull/9299/files#diff-51a114fecaac13cd41162f4c956e7508R48).